Makefile

```diff
@@ -534,7 +534,10 @@ cpplint:
 	--exclude_path src/operator/contrib/ctc_include
 
 pylint:
-	pylint --rcfile=$(ROOTDIR)/ci/other/pylintrc --ignore-patterns=".*\.so$$,.*\.dll$$,.*\.dylib$$" python/mxnet tools/caffe_converter/*.py
+	python3 --version  # Python 3.5.2
```

Comment:
What if the machine has not installed one of python3 or python2?

Reply:
A developer on such a machine will be unable to see syntax errors that they may be creating:
* __print "Hi"__ is valid Python 2 but is a syntax error in Python 3
* __async def func():__ is valid Python 3 but is a syntax error in Python 2

Reply:
I think it is fine to expect that people install both versions.

Reply:
I understand that checking on both versions is necessary and I'm supportive of it, but I've seen some machines images that came with no python at all and people may just randomly install only one of the python versions. To ensure an out-of-box experience for our users we should either clearly specify installation of both python versions as a prerequisite or we do it for them within the script.

Reply:
Not all USERS...  All Developers/Contributors who want to run __make pylint__.

Users are free to run mxnet on either Python or on legacy Python -- they do not need both.

Reply:
I would claim that developers are users of all the tools we provide, including this Makefile.

Reply:
What is the ASK?

Reply:
Sorry, but what does ASK mean here?

Reply:
This PR no longer requires Contributor to have multiple versions of Python installed.